obs-packaging/runtime/update.sh

```diff
@@ -92,10 +94,13 @@ replace_list=(
 	"kata_shim_version=${SHIM_REQUIRED_VERSION}"
 	"ksm_throttler_version=${KSM_THROTTLER_REQUIRED_VERSION}"
 	"linux_container_version=${KERNEL_REQUIRED_VERSION}"
-	"qemu_lite_version=${KATA_QEMU_LITE_REQUIRED_VERSION}"
 	"qemu_vanilla_version=${KATA_QEMU_VANILLA_REQUIRED_VERSION}"
 )
 
+if [ "$GO_ARCH" == "amd64" ]; then
+        replace_list+=" qemu_lite_version=${KATA_QEMU_LITE_REQUIRED_VERSION}"
```

Comment:
In bash, when adding elements to an array,  you need to surround such elements with `(` `)`. 

For some other shells that's not needed, e.g. on zsh